README.md

```diff
@@ -10,7 +10,7 @@ SPDX-License-Identifier: Apache-2.0
 
 In addition to `sbomnix` this repository is a home to [nixgraph](./doc/nixgraph.md), a python library and command line utility for querying and visualizing dependency graphs for [nix](https://nixos.org/) packages.
 
-For a demonstration of how to use `sbomnix` generated SBOM in automating vulnerability scans see: [vulnxscan](scripts/vulnxscan/README.md#example-automating-nix-vulnerability-scans).
```

Comment:
..., see...

Reply:
Fixed in https://github.com/tiiuae/sbomnix/commit/b424d9b917e0de3151289f417ef927f3726e339a